pages/CONTRIBUTING-docs.md

```diff
@@ -0,0 +1,75 @@
+
+## Welcome
+Thank you for helping us improve our documentation.
+
+This guide is intended for use by documentation contributors and will help you get started with the basics of our working processes.
+
+## Before you start
+To contribute to the JSON Schema documentation you will need to complete the following steps:
+
+1. [Create a GitHub account](https://docs.github.com/en/get-started/start-your-journey/creating-an-account-on-github), if you don't have one already.
+2. Read our [Code of Conduct](https://json-schema.org/overview/code-of-conduct).
+3. [Install the JSON Schema website](https://github.com/json-schema-org/website/blob/main/INSTALLATION.md) on your computer.
+4. Read our style guide. 
+
+## Community engagement
+To stay updated with news about the JSON Schema project and its documentation, engage with our team, ask questions, and discuss ideas, please join our [Slack workspace](https://json-schema.org/slack). We particularly recommend the following channels:
+
+- #documentation: Discussions related to the JSON Schema documentation.
+
+- #website: Discussions related to our website.
+
+- #introductions: Self-introductions and connecting with other members of the community.
+
+- #community-announcements: Events and updates related to the JSON Schema project.
+
+- #anouncements: Important announcement from the JSON Schema organization.
+
+- #stack-overflow: Stack overflow questions tagged with JSON Schema.
+  
+## Best practices
+
+This is a list of best practices the JSON Schema project strives for and resources we recommend for additional information:
+
+1. File and branching naming conventions: We use dashes instead of spaces, and lowercase letters to name .md files and branches. For example: `implementers-guide.md`, `web-feat-diataxis`.
+2. Commit messages: To learn how to write good commit messages, see [Step-by-step guide on how to write good commit messages on Git](https://www.freecodecamp.org/news/how-to-write-better-git-commit-messages/).
+
+## Contribution workflow 
+
+Here we describe three ways you can help us improve our documentation:
+
+
+1. Review pull requests
+2. Work on an existing issue 
+3. Propose changes 
+
+### Review pull requests
+
+Are you new to the JSON Schema community? Help us review pull requests. Reviewing pull requests helps you familiarize yourself with our workflows, learn how others contribute, and understand the project's needs and goals. It's a great way to start contributing and you'll be better prepared to make your own contributions.  
+
+### Work on an existing issue
+
+To work on an existing issue, go to the [JSON Schema docs board](https://github.com/orgs/json-schema-org/projects/16) and check all the open issues.  
+
+If the issue you select has no assignees, you can claim the issue by assigning it to yourself and tag @benjagm in the comments to let us know you'll be working on it. 
```

Comment:
Instead to tagging me let's ask contributors to tag "Docs Team" @json-schema-org/docs-team